#!/usr/bin/env python
PACKAGE = "image_processing"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("H_MIN", int_t, 0, "Hue min val", 0, 0, 256)
gen.add("S_MIN", int_t, 0, "Saturation min val", 0, 0, 256)
gen.add("V_MIN", int_t, 0, "Value min val", 0, 0, 256)
gen.add("H_MAX", int_t, 0, "Hue max val", 256, 0, 256)
gen.add("S_MAX", int_t, 0, "Saturation max val", 256, 0, 256)
gen.add("V_MAX", int_t, 0, "Value max val", 256, 0, 256)

exit(gen.generate(PACKAGE, "image_processing", "hsvThresh"))
